### PR TITLE
[stable/sonatype-nexus]: added support for service account

### PR DESCRIPTION
**Pull request by emas80**
#### What this PR does / why we need it:

Add the possibility to specify an existing Kubernetes service account, or let the system to create one automatically.
This allows to give specific permissions to the Kubernetes service account used by Nexus instead of using the default service account.

In particular, it makes simpler to give the permissions in order to be able to upload the backup to GCP.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/sonatype-nexus/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: sonatype-nexus
-version: 1.21.3
+version: 1.21.4
```
Comment (rjkernick) [inline stable/sonatype-nexus/Chart.yaml]: Please bump the minor version as this adds features

**Review by rjkernick — CHANGES_REQUESTED** [review]

**Comment by rjkernick** [comment]
Hello, all commits need signed in order for the pull request to be merged into the repository: https://github.com/helm/charts/pull/20032/checks?check_run_id=383520783

**Comment by emas80** [comment]
Hi @rjkernick , sorry for the delay I did not get any notification from GitHub :-(
I've signed the commits and fixed the minor version.

**Comment by rjkernick** [comment]
One last item. Merged in a PR in front of you, please merge in the latest master and increment the chart version if needed. Don't forget to sign the commits. Thanks!

**Pull request merged**